gum/backend-x86/gumstalker-x86.c

```diff
@@ -103,27 +105,58 @@ enum
 {
   PROP_0,
   PROP_IC_ENTRIES,
+  PROP_ADJ_BLOCKS,
 };
 
 struct _GumStalker
 {
   GObject parent;
 
   guint ic_entries;
+  /*
+  * Stalker compiles each block on demand. However, in when we reach the end of a
```

Comment:
Off by one space here and below. (Should be careful to avoid going > 80 cols when fixing this.)